triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -309,6 +331,9 @@ object Server {
                     triggerInstances => complete(successResponse(triggerInstances))
                   )
             }
+          },
+          pathPrefix("v1" / "status" / JavaUUID) { uuid =>
```

Comment:
Since we're returning the whole history here, would it make more sense to call this endpoint `history` or `log`?

Reply:
I have thought about that. Don't want to get analysis paralysis on something so trivial. I think status is the right word (even acknowledging that the entire history is returned).